transformer_engine/pytorch/module/base.py

```diff
@@ -113,6 +113,7 @@ def add_ub(
         num_splits: int = 0,
         aggregate: int = 0,
         atomic_gemm: int = 0,
+        use_ce: int = 1,
```

Comment:
We're a bit inconsistent about the type of `use_ce`. It's `bool` in `tex.UbufP2PCommOverlap`, so it seems appropriate for it to be `bool` here:
```suggestion
        use_ce: bool = True,
```